libs/Transisthor/TransisthorECSLogic/Client/Systems/SendToServer.hpp

```diff
@@ -42,6 +42,7 @@ struct SendToServer : public ecs::System {
         (void)networkId;
```

Comment:
Because add this point (when the types list is empty), it must do nothing.
It's used in a recursive case.
The vois is in order to have none unused variables.